server/models/user.js

```diff
@@ -32,12 +33,17 @@ const User = {
       }
       return String(role);
     },
+    dailyMessageLimit: (dailyMessageLimit = null) => {
+      return dailyMessageLimit === null ? null : Number(dailyMessageLimit);
+    },
```

Comment:
Should be >= 1

Reply:
For this, we use the `dailyMessageLimit` field to store if the option is enabled/disabled and the message limit. The way it currently works is if the option is off, it sets the field to `NULL` but when it is enabled we set it to the int value.